### PR TITLE
Fix #11554, by checking the return code of r_bin_mdmp_init_directory_…

### LINKED ISSUE #11554: Issue 10403 in oss-fuzz: radare2/ia_fuzz: Timeout in radare2_ia_fuzz
[clusterfuzz-testcase-minimized-ia_fuzz-5756355915284480.dms.zip](https://github.com/radare/radare2/files/2387911/clusterfuzz-testcase-minimized-ia_fuzz-5756355915284480.dms.zip)

### PR DESCRIPTION
**Pull request by mscherer**
…entry

### CODE REVIEW

File: libr/bin/format/mdmp/mdmp.c [hunk 1]
```diff
@@ -720,7 +720,9 @@ static bool r_bin_mdmp_init_directory(struct r_bin_mdmp_obj *obj) {
 		ut32 delta = i * sizeof (struct minidump_directory);
 		int r = r_buf_read_at (obj->b, rvadir + delta, (ut8*) &entry, sizeof (struct minidump_directory));
 		if (r) {
-			r_bin_mdmp_init_directory_entry (obj, &entry);
+			if (!r_bin_mdmp_init_directory_entry (obj, &entry)) {
+				return false;
+			};
```
Comment (radare) [inline libr/bin/format/mdmp/mdmp.c]: unnecessary ;

**Comment by radare** [comment]
so much xx

**Comment by mscherer** [comment]
Ok so it seems that the tests have some unsupported enumeration, so I guess the right fix is either to ignore them (but that's a hack), or to add support for it. I think support is cleaner, but then, what about future format change, do we want to declare a file using a unsupported enumeration valid or not ?

**Comment by XVilka** [comment]
See also https://github.com/radare/radare2/issues/10992
see `om` output that all maps do not even have "read" attribute, after setting it everything works.

**Pull request merged**